src/main/java/com/ays/backend/user/model/ERole.java

```diff
@@ -0,0 +1,7 @@
+package com.ays.backend.user.model;
+
+public enum ERole {
+    ROLE_VOLUNTARY,
+    ROLE_WORKER,
```

Comment:
Please add values to these enums, as you'll have indices instead, when you want to get the value of it.